tardis/apps/publication_workflow/static/publication-form/publication_form.js

```diff
@@ -0,0 +1,504 @@
+// TODO: remove hard coded URLs in case urls.py changes
+
+// A filter used to remove items from the list of available datasets
+// when added to the list of datasets to include in the publication
+/*eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/
+angular
+.module('publicationWorkflow')
+.filter('removeMatchingDatasets', function () {
+    return function (input, compareTo) {
+        if (angular.isUndefined(input)) {
+            return input;
+        }
+        var output = [];
+        for (var i = 0; i < input.length; i++) {
+            var add = true;
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 16 spaces but found 12.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                var add = true;
```